zeq/hbp/vocabulary.h

```diff
@@ -23,9 +36,46 @@ static const uint128_t EVENT_SELECTED_IDS(
     lunchbox::make_uint128( "hbp::SelectedIDsEvent" ));
 static const uint128_t EVENT_TOGGLE_ID_REQUEST(
     lunchbox::make_uint128( "hbp::ToggleIDRequest" ));
+static const lunchbox::uint128_t EVENT_REQUEST(
+    lunchbox::make_uint128( "hbp::RequestEvent" ));
+static const lunchbox::uint128_t EVENT_IMAGE_RAW(
+    lunchbox::make_uint128( "hbp::imageRawRGBAEvent" ));
 //@}
 
 /**
+ * Serialize the given rgba raw image into an Event of type EVENT_IMAGE_RAW.
+ * The image format must be raw rgba with 8bits per component.
+ * @param image the image.
+ * @return the serialized event.
+ */
+::zeq::Event serializeimageRawRGBA( const imageRawRGBA& image );
+
+/**
+ * Deserialize the given EVENT_IMAGE_RAW event into an rgba raw image.
+ * The image format is raw rgba with 8bits per component.
```

Comment:
Add a note that the data pointer in the Image is only valid as long as the Event stays alive.